packages/cloudcasa/overlay/questions.yaml

```diff
@@ -0,0 +1,13 @@
+namespace: cloudcasa-io
+labels:
+  io.rancher.certified: partner
+categories:
+- Application
```

Comment:
Read my other comment first.
If you decide to keep the files in the overlay directory here, please remove these.